packages/slate-simulator/rollup.config.js

```diff
@@ -0,0 +1,65 @@
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import uglify from 'rollup-plugin-uglify'
+import sourcemaps from 'rollup-plugin-sourcemaps'
+import pkg from './package.json'
+
+// UMD build for browsers
+const umdConfig = {
```

Comment:
Is there a better way for us to share all of these configuration logic somewhere, instead of having to duplicate it in each new monorepo package here? It seems like it could be a few util functions in a top-level helper I think.

Reply:
I looked at this a bit and I'm not sure. The problem is that despite all the similarities, there's a decent amount of variance for the different config files:

* the output name
* the UMD globals
* the dependencies to be considered `external` both for UMD and CJS/ES
* the plugins (a few of them need `alias` or `replace`, for example, but not all)

I didn't see a great way to condense the configuration logic. I think the abstraction is worse than what I've got here, because this is clearly comprehendible as normal rollup options, whereas I'd have weird ad-hoc exports/configuration to try to extract out shared bits.